src/components/CompareResults/CompareResultsViewInit.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from 'react';
+
+import { useParams } from 'react-router-dom';
+
+import { repoMap } from '../../common/constants';
+import { useAppDispatch, useAppSelector } from '../../hooks/app';
+import useFetchCompareResults from '../../hooks/useFetchCompareResults';
+import { setSelectedRevisions } from '../../reducers/SelectedRevisions';
+import { fetchCompareResults } from '../../thunks/compareResultsThunk';
+import { fetchSelectedRevisions } from '../../thunks/selectedRevisionsThunk';
+
+// component to fetch set selectedRevs revisions when Compare View is loaded
+function CompareResultsViewInit() {
```

Comment:
The variable names are a bit confusing to me. Maybe put the type in their names for a better understanding, eg: revisionList vs revisionCommitList vs revisionsAsString